string label
int32 class_id
float32 confidence
float32 x_min
float32 y_min
float32 x_max
float32 y_max
